source/index.html.erb

```diff
@@ -156,8 +156,7 @@ layout: govuk_wide
           <li>
             <a
```

Comment:
Both this link and the next one also need `govuk-link` classes 👀

Reply:
Good spot!